src/app/chat/views/communities.nim

```diff
@@ -228,6 +247,16 @@ QtObject:
   proc addCommunityToList*(self: CommunitiesView, community: var Community) =
     var communities = @[community]
     community = self.populateChats(communities)[0]
+
+    #echo "COMMUNITY CHATS: ", $community.chats
+
+    for cat in community.categories:
+      echo "Category: ", cat.name
+
+      for c in community.chats:
+        if c.categoryId == cat.id:
+          echo "-> ", c.name, ": ", c.position
```

Comment:
This is just debugging output which can be ignored and will be removed before merge.

Helps inspecting what chat items belong to which categories when we get an update from status-go.

That's how I verified the bug reported in https://github.com/status-im/status-go/issues/2376